### PR TITLE
Ver.2.0.1

### PR DESCRIPTION
**Pull request by YaroslavHalkaEPAM**

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -7,32 +7,38 @@ module.exports = function(grunt) {
...
     grunt.initConfig({
-        clean:{
+        clean: {
             all:{
                src: ['public/*']
             }
         },
+        sprite:{
+             all: {
+                src: 'src/images/png/*.png',
```
Comment (YaroslavHalkaEPAM) [inline Gruntfile.js]: The name of folder should be sprite, not png

File: Gruntfile.js [hunk 2]
```diff
@@ -7,32 +7,38 @@ module.exports = function(grunt) {
...
+                dest: 'src/images/spritesheet.png',
```
Comment (YaroslavHalkaEPAM) [inline Gruntfile.js]: The file name should be sprite.png

File: Gruntfile.js [hunk 3]
```diff
@@ -7,32 +7,38 @@ module.exports = function(grunt) {
...
+                destCss: 'src/images/sprites.css'
```
Comment (YaroslavHalkaEPAM) [inline Gruntfile.js]: Destination should be scss folder

File: Gruntfile.js [hunk 4]
```diff
@@ -61,9 +75,18 @@ module.exports = function(grunt) {
                         dest: 'public/templates/',
                         filter: 'isFile'      
                     },
+                     {  expand: true,
```
Comment (YaroslavHalkaEPAM) [inline Gruntfile.js]: you should skip sprite src folder

**Comment by YaroslavHalkaEPAM** [comment]
main.css why this file is in git?

**Comment by YaroslavHalkaEPAM** [comment]
src/images/026.png why this file is in git?

**Comment by YaroslavHalkaEPAM** [comment]
src/images/png/2.png file name is not informative

**Comment by YaroslavHalkaEPAM** [comment]
src/images/sprites.css why this file is in git?

**Comment by YaroslavHalkaEPAM** [comment]
src/images/spritesheet.png why this file is in git?

File: Gruntfile.js (hunk 1 above)
Comment (al6mina) [inline Gruntfile.js]: done

File: Gruntfile.js (hunk 2 above)
Comment (al6mina) [inline Gruntfile.js]: done

**Comment by al6mina** [comment]
DELETE 
src/images/026.png,
src/images/spritesheet.png,
main.css

CHANGE
src/images/png/2.png file name

**Pull request merged**